benchmarks/util.py

```diff
@@ -76,8 +76,15 @@ def is_xla_device_available(devkind):
   return r.returncode == 0
 
 
-def move_to_device(item, device):
-  return pytree.tree_map_only(torch.Tensor, lambda t: t.to(device), item)
+def move_to_device(item, device, use_torch_xla2: bool = False):
+  if use_torch_xla2:
+    import torch_xla2
+    import jax
+    move_to_device_func = lambda t: jax.device_put(
+        torch_xla2.tensor.t2j(t), device)
```

Comment:
Is this going to work if `device="cpu"`?

Reply:
Just checked, CPU also works.